openmc/stats/multivariate.py

```diff
@@ -616,6 +621,128 @@ def from_xml_element(cls, elem):
         origin = [float(x) for x in elem.get('origin').split()]
         return cls(r, phi, z, origin=origin)
 
+class MeshSpatial(Spatial):
+    """Spatial distribution for a mesh.
+
+    This distribution specifies a mesh to sample over, chooses whether it will
+    be volume normalized, and can set the source strengths.
+
+    .. versionadded:: 0.13.3
+
+    Parameters
+    ----------
+    mesh : openmc.MeshBase
+        The mesh instance used for sampling, mesh is written into settings.xml,
+        mesh.id is written into the source distribution
+    strengths : Iterable of Real, optional
+        A list of values which represent the weights of each element. If no source
+        strengths are specified, they will be equal for all mesh elements.
+    volume_normalized : bool, optional
+        Whether or not the strengths will be normalized by volume. Default is True.
```

Comment:
For clarity as in line 675:

```suggestion
        Whether or not the strengths will be normalized (multiplied) by volume. Default is True.
```

Reply:
Hrmmm yeah I think "multiplied" is simply more correct here. Updating now.